### PR TITLE
Have --proxy-to-worker more directly alias PROXY_TO_WORKER setting. NFC

### PR DESCRIPTION
**Pull request by sbc100**
This avoids having two ways check for this option which can be
confusing and error prone.

### CODE REVIEW

**Review by aheejin — APPROVED** [review]
Should we remove this too? (I disabled the auto-merge)
https://github.com/emscripten-core/emscripten/blob/b61ff56fe215bee34f174cb2c89cc2b66356bf57/emcc.py#L256

**Comment by sbc100** [comment]
> Should we remove this too? (I disabled the auto-merge)
> https://github.com/emscripten-core/emscripten/blob/b61ff56fe215bee34f174cb2c89cc2b66356bf57/emcc.py#L256

Nice catch!   Thanks for spotting that.

**Comment by sbc100** [comment]
> > Should we remove this too? (I disabled the auto-merge)
> > https://github.com/emscripten-core/emscripten/blob/b61ff56fe215bee34f174cb2c89cc2b66356bf57/emcc.py#L256
> 
> Nice catch! Thanks for spotting that.

Wait.. this change does remove that line doesn't it?

File: emcc.py [hunk 1]
```diff
@@ -253,7 +253,6 @@ def __init__(self):
     self.memory_init_file = None
     self.use_preload_cache = False
     self.use_preload_plugins = False
-    self.proxy_to_worker = False
```
Comment (sbc100) [inline emcc.py:256]: Here

Comment (aheejin) [reply emcc.py:256]: Oh sorry, I missed it.

**Pull request merged**